tasks/run_cd4pe_job.json

```diff
@@ -24,15 +24,15 @@
     },
     "docker_image": {
```

Comment:
I'm leaving the parameter names the same even though they have "docker" in them to make this not represent a breaking change.  That said, these parameters are not really used by customers, just by our cd4pe_deployments module when invoking jobs. So changing them and causing the additional cost of forcing customers to align their versions doesn't seem worth it.